modules/check_in/spec/request/v2/sessions_request_spec.rb

```diff
@@ -158,6 +160,20 @@
         expect(response.status).to eq(200)
         expect(JSON.parse(response.body)).to eq(service_resp[:permission_data])
       end
+
+      context 'with service_refactor feature flag on' do
+        it 'returns a success response' do
+          allow(Flipper).to receive(:enabled?)
+            .with('check_in_experience_services_refactor').and_return(true)
+          allow_any_instance_of(::V2::Lorota::Service).to receive(:token).and_return(service_resp)
+          expect_any_instance_of(::V2::Lorota::Service).to receive(:token).once
```

Comment:
Same as above comments

Reply:
addressed in 6ea726dd366a3daf29ab012fc161ed7dd6a090f7

Reply:
Could you please move these lines inside before block as well?

Reply:
addressed in c9bbe5fbcc9d3bdba975960a72c9067777e42886